test/UserProduct.spec.js

```diff
@@ -47,3 +50,73 @@ describe('UserProduct tests', () => {
     expect(wrapper.find('.service-info p').text()).toBe(testProduct.text)
   })
 })
+
+describe('User Product helper functions tests', () => {
+  test('should check for PPR and MHR products', () => {
+
+    let products = [
+      {
+        "description": "Business Registry & Name Request",
+        "code": "BUSINESS",
+      },
+      {
+        "description": "Personal Property Registry",
+        "code": "PPR",
+      },
+      {
+        "description": "Manufactured Home Registry",
+        "code": "MHR",
+      },
+      {
+        "description": "Wills Registry",
+        "code": "VS"
+      }
+    ]
+
+    expect(hasMhrAndPprProducts(products)).toBe(true)
+
+    products = [
+      {
+        "description": "Business Registry & Name Request",
+        "code": "BUSINESS",
+      },
+      {
+        "description": "Personal Property Registry",
+        "code": "PPR"
+      }
+    ]
+
+    expect(hasMhrAndPprProducts(products)).toBe(false)
+
+  })
+
+  test('should combine MHR and PPR products into one registry (asset)', () => {
+
+    const products = [
+      {
+        "description": "Business Registry & Name Request",
+        "code": "BUSINESS",
+      },
+      {
+        "description": "Personal Property Registry",
+        "code": "PPR",
+      },
+      {
+        "description": "Manufactured Home Registry",
+        "code": "MHR",
+      },
+      {
+        "description": "Wills Registry",
+        "code": "VS"
+      }
+    ]
+
+    const combinedProducts = combineAssetProducts(products)
+    const mhrPprProduct = combinedProducts.find(p => p.code === ProductCode.PPR_MHR)
+
+    expect(combinedProducts.length).toBe(3)
+    expect(mhrPprProduct).not.toBe(null)
```

Comment:
There is "toBeNull()" that you can use here :)

Reply:
Refactored